Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -56,6 +56,7 @@ static void InitAVCodec()
   if (first_run)
   {
     av_register_all();
+    avformat_network_init();
```

Comment:
Does this also include devices like Chromecast/Miracast (assuming they have some sort of standard interface there)? And, could this be a potential extension point to plug in RTMP/HLS streaming?
(Not asking you to do it, but I know someone thats potentially interrested in going there)